packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.test.ts

```diff
@@ -2952,6 +2952,7 @@ describe('SavedObjectsRepository', () => {
           namespaces: [namespace ?? 'default'],
           coreMigrationVersion: expect.any(String),
           typeMigrationVersion: '1.1.1',
+          managed: false,
```

Comment:
There isn't a way to set `managed` in the current PR, only the default is applied. Personally, working in this test suite is a nightmare but I'll try to give it a shot if you feel strongly about it. (otherwise, we can leave that up to the follow up PR and get the mappings updated now)